admin/banner/class-admin-banner-sidebar.php

```diff
@@ -88,45 +88,19 @@ protected function get_premium_spot() {
 
 		$premium_spot->set_extra(
 			/* translators: %1$s expands to the plugin name */
-			'<h2>' . sprintf( __( 'Get %1$s', 'wordpress-seo' ), 'Yoast SEO Premium' ) . '</h2>' .
+			'<h2>' . sprintf( __( 'Upgrade to %1$s', 'wordpress-seo' ), 'Yoast SEO Premium' ) . '</h2>' .
```

Comment:
Suggestion to use 'esc_html_' here and in the lines below instead of '__',  so translators cannot accidentally insert html.